common/djangoapps/entitlements/models.py

```diff
@@ -1,8 +1,82 @@
 import uuid as uuid_tools
+from datetime import datetime, timedelta
 
+import pytz
 from django.conf import settings
+from django.contrib.sites.models import Site
 from django.db import models
+
 from model_utils.models import TimeStampedModel
+from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
+
+
+class CourseEntitlementPolicy(models.Model):
+    """
+    Represents the Entitlement's policy for expiration, refunds, and regaining a used certificate
+    """
+
+    expiration_period_days = models.IntegerField(
+        default=450,
+        help_text="Number of days from when an entitlement is created until when it is expired.",
+        null=False
+    )
+    refund_period_days = models.IntegerField(
+        default=60,
+        help_text="Number of days from when an entitlement is created until when it is no longer refundable",
+        null=False
+    )
+    regain_period_days = models.IntegerField(
```

Comment:
Would https://docs.djangoproject.com/en/1.8/ref/models/fields/#positiveintegerfield be a better choice here?